cdap-proto/src/main/java/co/cask/cdap/proto/audit/AuditMessage.java

```diff
@@ -26,6 +26,8 @@
  */
 @Beta
 public class AuditMessage {
+  private final int version = 1;
```

Comment:
Might as well be static, right?
Since all instances of this class will have the same value.

Upside - save memory (or maybe compiler will automatically already do this).
Any downside?